src/status_im/ui/screens/home/views/inner_item.cljs

```diff
@@ -98,9 +108,17 @@
      ;; won't be applied correctly.
      (first @(re-frame/subscribe [:contacts/contact-two-names-by-identity chat-id])))])
 
+(defn highlight-chat? [home-item]
+  (let [{:keys [group-chat unviewed-messages-count last-message highlight public?]} home-item
+        {:keys [content-type community-id]} last-message]
+    (if (> unviewed-messages-count 0)
+      (or highlight (content-type-community-invite? content-type community-id))
```

Comment:
why can't we highlight always when the highlight is true?  i mean why status-go doesn't return final highlight value?

Reply:
Yes, that sounds much better, I will move this to status-go👍 Thanks

Reply:
Done 👍